src/signer.ts

```diff
@@ -50,6 +58,18 @@ export class InMemorySigner extends Signer {
         await this.keyStore.setKey(networkId, accountId, keyPair);
         return keyPair.getPublicKey();
     }
+    
+    /**
+     * Creates a public key for the account, network and keyPair provided
+     * @param networkId The targeted network. (ex. default, betanet, etc…)
+     * @param accountId The NEAR account to assign a public key to
+     * @param keyPair The keyPair to use for signing
+     * @returns {Promise<PublicKey>}
+     */
+    async fromKeyPair(networkId: string, accountId: string, keyPair: KeyPair): Promise<PublicKey> {
```

Comment:
@mattlockyer sorry that missed it the first time, but I think following API makes most sense:

`InMemorySigner.fromKeyPair(networkId: string, accountId: string, keyPair: KeyPair): Promise<Signer>`

This is useful every time we need to have throwaway signer. 

Basically create signer instance as a whole from single KeyPair. Generally this only makes sense for in-memory signer, i.e. it won't work for Ledger, etc.